ui/app/src/state/epics/pathNavigator.ts

```diff
@@ -121,7 +121,7 @@ export default [
            },
            state
          ]) => {
-          if (response?.length > 0) {
+          if (response?.length > 0 || type === pathNavigatorSetCollapsed.type) {
             localStorage.setItem(
```

Comment:
don't we have a util function to set this item?

Reply:
not yet